tests/workflows/protocols/pw/test_relax.py

```diff
@@ -65,10 +65,50 @@ def test_spin_type(fixture_code, generate_structure):
         assert parameters['SYSTEM']['starting_magnetization'] == {'Si': 0.1}
 
 
-@pytest.mark.parametrize('relax_type', RelaxType)
-def test_relax_type(fixture_code, generate_structure, relax_type):
-    """Docs."""
+def test_relax_type(fixture_code, generate_structure):
+    """Test ``PwRelaxWorkChain.get_builder_from_protocol`` with ``spin_type`` keyword."""
     code = fixture_code('quantumespresso.pw')
     structure = generate_structure()
-    builder = PwRelaxWorkChain.get_builder_from_protocol(code, structure, overrides={'relax_type': relax_type.value})
-    assert builder.relax_type.value == relax_type.value
+
+    builder = PwRelaxWorkChain.get_builder_from_protocol(code, structure, relax_type=RelaxType.NONE)
+    assert builder.base['pw']['parameters']['CONTROL']['calculation'] == 'scf'
+    with pytest.raises(KeyError):
+        builder.base['pw']['parameters']['CELL']  # pylint: disable=pointless-statement
```

Comment:
I think the following type of assertions may be more intuitive and cleaner written as
```suggestion
    assert 'CELL' not in builder.base['pw']['parameters']
```

Reply:
Haha, of course, that's a little less convoluted. 😅 I'll fix it!

Reply:
Actually, seems I made a mistake in the tests (e.g. for `ATOMS_VOLUME` the parameters should have a `CELL` input), but the code still raised a `KeyError` because `builder.base['pw']['parameters']` is a `Dict`.